### PR TITLE
Oluchi/MarketPlace

### PR DESCRIPTION
**Pull request by oluchicharity**

### CODE REVIEW

File: SmartContract/contracts/PharmX.sol [hunk 1]
```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract PharmX {
```
Comment (udodinho) [inline SmartContract/contracts/PharmX.sol]: This should be the name of your contract and the file name `MedicineMarketplace.sol`

File: SmartContract/contracts/PharmX.sol [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        bool isPrescriptionRequired;
+    }
+
+    uint256 public nextMedicationId = 1;
+    uint256 public platformFeePercent = 2; 
+    address public platformAdmin;
+
+    mapping(uint256 => Medication) public medications;
+
+    event MedicationCreated(uint256 id, string productName, uint256 pricePerUnit, uint256 stockQuantity, address owner);
```
Comment (udodinho) [inline SmartContract/contracts/PharmX.sol]: Pls, just indexed one or two of the event parameters

File: SmartContract/contracts/PharmX.sol (hunk 1 above)
Comment (oluchicharity) [reply SmartContract/contracts/PharmX.sol]: alright

File: SmartContract/contracts/PharmX.sol (hunk 2 above)
Comment (oluchicharity) [reply SmartContract/contracts/PharmX.sol]: alright

**Pull request merged**